packages/bpk-component-floating-notification/src/BpkFloatingNotification-test.js

```diff
@@ -104,4 +104,13 @@ describe('BpkFloatingNotification', () => {
 
     expect(onClick).toHaveBeenCalledTimes(1);
   });
+
+  it('should do function when unmounted (exited)', () => {
+    const onExit = jest.fn();
+    render(<BpkFloatingNotification onExit={onExit} {...props} />);
+
+    setTimeout(() => {
+      expect(onExit).toHaveBeenCalledTimes(1);
+    });
```

Comment:
Without the `setTimeout()` this test fails. I beileve it's because we need to wait for the notification to disappear (it takes ~4800ms, 800ms for the in and out animations and it holds for 4 seconds), however it is odd that I don't have to specify a period of time to wait.

I'm open to suggestions on improving this, but it appears to work.